nengo_bones/templates/setup.cfg.template

```diff
@@ -96,9 +96,24 @@ allclose_tolerances =
 {% endfor %}
 {% endif %}
 
+[pydocstyle]
+ignore =
+    D10,
+    D202,
+    D203,
+    D213,
+    D402,
```

Comment:
Probably also want `D401`, which is the imperative mood one. Also, maybe we should put comments after each of these lines with a word or two saying what it is.

Reply:
The commit message of d2ce6d5ffca0eca0adf488384e75505615924733 gives my thoughts on D401

Reply:
And I'd rather just link to http://www.pydocstyle.org/en/3.0.0/error_codes.html ... we don't gives description for flake8, which is similarly code-based

Reply:
It does take a bit of work to cross-reference the two lists, but not many people are going to be editing this, so it's not a big deal to me.

Reply:
Also, D415 and D416 are missing from that list? They need to update their docs!

Reply:
Yeah they actually have a problem with their current docs right now as they don't render the table: https://github.com/PyCQA/pydocstyle/issues/380